UI-Kit/README.md

```diff
@@ -0,0 +1,41 @@
+# UI Kit
+
+To provide reference design materials and standardize elements across the app, we created a basic UI kit initially consisting of four pages in an updated Figma file.
+
+The four pages are palettes (colors), typography (fonts), buttons, and grids.
+
+### Colors / Palettes
+
+<img title="Colors-Palettes" alt="A Figma page with a row of six colors swatches and a white box with usage information for each color below" src="./Colors-Palettes.png">
+
+Here's the colors / palettes page. We added six color swatches matched to the exact colors from the current UD marketing website, with one exception. The primary UD logo orange did not pass all contrast tests for accessibility, so we created a new darker shade labeled "high-contrast" that has sufficient contrast for all use cases.
+
```

Comment:
This particular palette is a great choice, and I like how organized everything is.

Reply:
I love the colors! Well contrasting and fit the purpose.